package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuclia-sync-agent-app",
-  "version": "1.2.2",
+  "version": "1.2.3",
   "description": "\"This is a Nuclia Sync Agent App\"",
```

Comment:
I know this is not coming from your PR, but why do we have double quotes wrapped in this description?